silo-vaults/contracts/incentives/VaultIncentivesModule.sol

```diff
@@ -0,0 +1,103 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity 0.8.28;
+
+import {Ownable2Step, Ownable} from "openzeppelin5/access/Ownable2Step.sol";
+import {EnumerableSet} from "openzeppelin5/utils/structs/EnumerableSet.sol";
+
+import {IVaultIncentivesModule} from "../interfaces/IVaultIncentivesModule.sol";
+import {IIncentivesClaimingLogic} from "../interfaces/IIncentivesClaimingLogic.sol";
+import {IIncentivesDistributionSolution} from "../interfaces/IIncentivesDistributionSolution.sol";
+
+/// @title Vault Incentives Module
+contract VaultIncentivesModule is IVaultIncentivesModule, Ownable2Step {
```

Comment:
let's add option to resque rewards, eg:

```
/// @dev in case we will get some unexpected reward, this is a way to unlock them, so they do not stuck here
    /// notice: this method should go to vault as well, as vault will be doing delegatecall, but it is no harm to
    /// have it here as well, it can not create any damage
    function rescueReward(address _token) {
        _token.transfer("all balance to vault");
        vault.immediateDistribution(balance);
    }
```

Reply:
The incentives module shouldn't receive rewards. Of course, someone can send any tokens to the module. But this is not a reason to add this kind of method. This module is a simple registry; it has no logic for reward claiming. Claiming will be performed by a Vault by delegating a call to the rewards claiming logic. It is hard for me to imagine that the VaultIncentivesModule, for some reason, will receive incentives.

Reply:
sure, should be enough to have it in vault.